### DISCUSSION TITLE
Show two information, or two commands on a single line

### DISCUSSION THREAD

**Discussion by ghost** [fastfetch-cli/fastfetch#1288] [Q&A] [answered]
Hello, I'm looking for someone's help if possible, I really can't do it with my own resources even after reading a bit the google.

```
"text": "ls -lact --full-time /etc |awk 'END {print $6}'", // Birth of your OS (Linux)
```
```
"text": "birth_install=$(stat -c %W /); current=$(date +%s); time_progression=$((current - birth_install)); days_difference=$((time_progression / 86400)); echo $days_difference days" , // Display days of OS (Installation)
```
I would like to merge both commands and appears in one line like:
**2024-03-16 (190 days)**

`Info - fastfetch 2.25.0 (x86_64)`

**Comment by CarterLi** [comment, marked as answer]
`birth_install=$(stat -c %W /); current=$(date +%s); time_progression=$((current - birth_install)); days_difference=$((time_progression / 86400)); echo $(ls -lact --full-time /etc |awk 'END {print $6}') $days_difference days`

**Comment by ghost** [comment]
Genius CarterLi. That's what i was looking for. Thank you!
